plugins/env/app/views/samson_env/_fields.html.erb

```diff
@@ -15,5 +15,6 @@
       </div>
     <% end %>
   <% end %>
+  <%= link_to "Add row", "#", class: "duplicate_previous_row" %> |
```

Comment:
there is no trailing at the right, the next line has whitespace, so it works out fine :)

![screen shot 2015-06-01 at 9 06 18 pm](https://cloud.githubusercontent.com/assets/11367/7928270/1e6673e4-08a2-11e5-96c2-69ba5493b8eb.png)